src/Tizen.NUI/src/public/Capture.cs

```diff
@@ -328,6 +328,14 @@ public string GenerateUrl()
             return url;
         }
 
+        /// Get Captured buffer.
+        /// </summary>
+        /// <returns>PixelBuffer of captured buffer</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public PixelBuffer GetCapturedBuffer()
+        {
+            return new PixelBuffer(Interop.Capture.GetCapturedBuffer(SwigCPtr), true);
+        }
```

Comment:
could you resolve the warnings?

src/Tizen.NUI/src/public/Capture.cs#L112
src/Tizen.NUI/src/public/Capture.cs#L118
src/Tizen.NUI/src/public/Capture.cs#L123
src/Tizen.NUI/src/public/Capture.cs#L144
src/Tizen.NUI/src/public/Capture.cs#L150
src/Tizen.NUI/src/public/Capture.cs#L332

Reply:
I fixed it thank you!